ui/wallet_page.go

```diff
@@ -228,8 +229,21 @@ func (pg *walletPage) getWalletMenu(wal *dcrlibwallet.Wallet) []menuItem {
 				textModal := newTextInputModal(common).
 					hint("Wallet name").
 					positiveButton(values.String(values.StrRename), func(newName string, tim *textInputModal) bool {
-						// todo handle error
-						pg.multiWallet.RenameWallet(wal.ID, newName)
+						tim.setError("") // clear error first
+
+						if err := pg.validateWalletName(newName); err != nil {
```

Comment:
This can be simplified.
```suggestion
						newName = strings.TrimSpace(newName)
						if name == "" {
```